drivers/renesas/rcar/console/rcar_console.S

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Need to use MULTI_CONSOLE_API for the console as the previous one is deprecated.

Reply:
can you check this again please? from the 2.0 release note I can read the following:

" All deprecated interfaces except the MULTI_CONSOLE_API have been removed from the code 
base."

Reply:
@soby-mathew I am not sure this API makes sense on the rcar  since rcar does not use any of the uarts available...and nobody will ever use the rcar uart. so could you maybe explain the benefits of this interface (and extra code) for platforms not using generic uarts please?

Reply:
Hi @ldts,
The old console API are deprecated in the sense they are marked for removal soon (see [here](https://github.com/ARM-software/arm-trusted-firmware/wiki/TF-A-Release-information#2removal-of-deprecated-interfaces)).  The expectation is all platforms will migrate to the new framework before removal.

There is a bit of extra code as a result of the new framework, but it allows piping the output to several console devices if required (in fact the console can be a memory buffer like the cbmem driver). It also allows easily switching the console depending on the state of the firmware like boot -> runtime -> crash.

We could merge the PR without the MUTLI_CONSOLE, but be aware that deprecated console is scheduled for removal soon.

Reply:
https://github.com/ARM-software/arm-trusted-firmware/wiki/TF-A-Release-information#removal-of-deprecated-interfaces

Reply:
ah awesome. great effort on the documentation btw